site/_posts/2021-03-16-securely-implement-oauth-vuejs.md

```diff
@@ -964,7 +1001,7 @@ When this final request is successful, we send all the data to our Vue client vi
 }
 ```
 
-The API key that we are passing in the `Authorization` HTTP header is not part of OAuth standard. You need it to call non-standard endpoints like the [User Registration API](https://fusionauth.io/docs/v1/tech/apis/registrations#retrieve-a-user-registration). We added this to show how you can use the API key if you decide to access endpoints protected by that key.
+> ***NOTE***: The API key that we are passing in the `Authorization` HTTP header is not part of OAuth standard. You need it to call non-standard endpoints like the [User Registration API](https://fusionauth.io/docs/v1/tech/apis/registrations#retrieve-a-user-registration). We added this to show how you can use the API key if you decide to access endpoints protected by that key.
```

Comment:
It feels like you want a generic callout style. We should talk about this. One easy path would be to pull it into an include file (perhaps `_callout.liquid`). This way we can do callouts in a consistent manner, just like we build image tags.